test/cli/cli.test.js

```diff
@@ -117,15 +119,6 @@ runCLITest('CLI', () => {
       .catch(done);
   });
 
-  it('unspecified host and port', (done) => {
-    testBin('')
-      .then((output) => {
-        expect(/http:\/\/localhost:[0-9]+/.test(output.stderr)).toEqual(true);
-        done();
-      })
-      .catch(done);
-  });
```

Comment:
We should not delete this tests, let's improve testing here

Reply:
Now we don't default host so we can change the assertion to false.

Reply:
host should be in IPv6 here

Reply:
I see `running at http:/` without specifying host and port -

![Screenshot at 2021-02-06 10-31-20](https://user-images.githubusercontent.com/46647141/107109474-d9811380-6866-11eb-96b5-5d75c8a74363.png)

Reply:
Bug :smile:

Reply:
We need to fix it, because it is very bad message

Reply:
Looking into it.

Reply:
I think we need to use `::` by default here.

Reply:
Ideally we should print message like `Project is running at https://server.com` or `Project is running at IPv4 (IPv6)`, it is in my roadmap, give me today to improve this

Reply:
Do we need an update here? 
https://github.com/webpack/webpack-dev-server/blob/b0868c399ee965727a97de303452b9841fd315e3/lib/Server.js#L592-L609

Reply:
Yes, we need improve it here

Reply:
```js
if (typeof hostname === 'undefined') { 
  prettyHostName = ?? // what should be printed here?
}
```

Reply:
hm, I need to look at code, why hostname here undefined :confused: